dotnet/Capstone/DAO/UserProfileSqlDao.cs

```diff
@@ -90,7 +90,7 @@ public UserProfile UpdateUserProfile(UserProfile profile, string username)
             {
                 sqlSet += ", users.skill_level = @skill_level";
             }
-            if (profile.FavoriteBird.id > 0)
+            if (profile.FavoriteBird.BirdId > 0)
```

Comment:
This is a good place to remind you to merge the updated database-rework into this branch. This specific line change may cause a small merge conflict, FYI.